### DISCUSSION TITLE
I get an error message but not sure what's wrong with it

### DISCUSSION THREAD

**Discussion by wsxwsx543** [pybind/pybind11#3306] [Q&A]
```
anaconda3\envs\lanedet\lib\site-packages\torch\include\pybind11\cast.h(1358): error: too few arguments for template template parameter "Tuple"
          detected during instantiation of class "pybind11::detail::tuple_caster<Tuple, Ts...> [with Tuple=std::pair, Ts=<T1, T2>]"
(1420): here

anaconda3\envs\lanedet\lib\site-packages\torch\include\pybind11\cast.h(1416): error: too few arguments for template template parameter "Tuple"
          detected during instantiation of class "pybind11::detail::tuple_caster<Tuple, Ts...> [with Tuple=std::pair, Ts=<T1, T2>]"
(1420): here

2 errors detected in the compilation of "/lanedet/lanedet/ops/csrc/nms_kernel.cu".
nms_kernel.cu
```
I am using VS2017 to compile this project.(https://github.com/Turoad/lanedet) I also tried different combination of versions of cudatoolkit. torch. torchvision but it still cannot work :(

**Comment by henryiii** [comment]
I don't think that's our fault - and you didn't even put what version of pybind11 this is using. The traceback isn't very helpful (as is often the case with NVCC - the line failing in nms_kernel.cu would be very helpful).  It is possible this was never designed to support windows, or it's very poorly tested (no CI in the repo), or something else. Seeing stdlib stuff on CUDA sounds suspect. I suspect it's a bug in lanedet.
